### PR TITLE
UIImage+QMUI 分类<qmui_imageWithGradientColors>渐变色方法增加左上至右下,右上至左下的倾斜的渐变色枚举

### PR DESCRIPTION
**Pull request by xixisplit**
倾斜的渐变色设计上也会经常用到.所以给qmui_imageWithGradientColors 增加了斜向的渐变色枚举

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/QMUI_iOS?pullRequest=1290) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/QMUI_iOS?pullRequest=1290) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/QMUI_iOS?pullRequest=1290) it.</sub>

File: QMUIKit/UIKitExtensions/UIImage+QMUI.h [hunk 1]
```diff
@@ -52,9 +52,10 @@ typedef NS_ENUM(NSInteger, QMUIImageResizingMode) {
 typedef NS_ENUM(NSInteger, QMUIImageGradientType) {
     QMUIImageGradientTypeHorizontal,
     QMUIImageGradientTypeVertical,
+    QMUIImageGradientTypeLeftTilt,
```
Comment (MoLice) [inline QMUIKit/UIKitExtensions/UIImage+QMUI.h:55]: 命名不直观，建议改为 QMUIImageGradientTypeTopLeftToBottomRight、QMUIImageGradientTypeTopRightToBottomLeft

Comment (xixisplit) [reply QMUIKit/UIKitExtensions/UIImage+QMUI.h:55]: 好的.之前我本地就是这个命名

**Comment by MoLice** [comment]
已发布 [4.4.0](https://github.com/Tencent/QMUI_iOS/releases/tag/4.4.0) 带上该提交。

**Pull request merged**